cnf-tests/testsuites/pkg/nodes/nodes.go

```diff
@@ -314,3 +314,11 @@ func nodeSelectorAsString(nodeSelector map[string]string) string {
 	}
 	return result
 }
+
+func IsSingleNode() (bool, error) {
```

Comment:
You either move this to the "cluster" package or rename the function as "IsSingleNodeCluster", otherwise it will read as nodes.IsSingleNode() from the client side